test/mapFactory.js

```diff
@@ -183,6 +183,22 @@
                     );
                 });
             },
+            pins: function (callback) {
+                require(["test/DataFactory", "src/map/ChoroplethStates", "src/map/Pins"], function (DataFactory, ChoroplethStates, Pins) {
+                    var pins = new Pins()
+                        .columns(DataFactory.GMap.simple.columns)
+                        .data(DataFactory.GMap.simple.data)
+                    ;
+                    var choro = new ChoroplethStates()
+                        .columns(["State", "Weight"])
+                        .data([["AL", 4779736], ["FL", 710231]])
```

Comment:
Yes - my rule of thumb is to include the data inline _unless_ it is needed in two diff tests, then I move it into the DataFactory.